articles/install-guide/standalone.md

```diff
@@ -30,8 +30,15 @@ To configure Visual Studio:
 1. Download and install [Visual Studio](https://visualstudio.microsoft.com/downloads/) 16.3 or greater, with the .NET Core cross-platform development workload enabled.
 2. Download and install the [Microsoft QDK](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit).
 
+To install the QDK for another environment, enter in the command line:
 
-## Develop with Q# using VS Code
+    dotnet new -i Microsoft.Quantum.ProjectTemplates
```

Comment:
@geduardo @cgranade @tcNickolas I'm getting some very odd behavior here. This line was giving me wild rendering problems when code blocked with triple ``'s, so I went to make it in-line with single quotes. Then, it renders as a full code block, but with the single quotes included. I remove the quotes entirely, and voila, it's rendering perfectly as a code block without any instructions telling it to do so. 

Is there some trick (or bug) in the markdown scheme that I'm missing here?

Reply:
I'm not sure why the ` ``` ` fences were failing, but indenting by four or more whitespace characters is an equivalent way of denoting literal code blocks, but comes with the disadvantage that you can't specify a language that way. The [DocFX Flavored Markdown reference](https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html?tabs=tabid-1%2Ctabid-a) has more detail.

(As a side note, you can now include cells from Jupyter Notebook files...?)